lib/twilio/device.ts

```diff
@@ -887,8 +890,7 @@ class Device extends EventEmitter {
    */
   private _destroyAudioHelper() {
     if (!this._audio) { return; }
-
-    this._audio.removeAllListeners();
+    this._audio._destroy();
```

Comment:
Is this why we didn't actually mark `_destroy` as a private function? Should we just remove the `_`?

Reply:
Correct. We cannot remove `_` because that implies it's a customer API.